test/common.js

```diff
@@ -0,0 +1,36 @@
+import co  from 'co';
+import jwt from 'jsonwebtoken';
+
+import config from '../src/config';
+
+const endpointPrefix = `/${config.get('version')}`;
+
+export function loginAsAdmin(server) {
+  return co(function*() {
```

Comment:
Here we could just use promise chains because it stays quite simple, still I think it's more readable like this.